site/app/templates/CreateNewAccount.twig

```diff
@@ -0,0 +1,88 @@
+{% set user_id_reqs = requirements["user_id"] %}
+{% set name_reqs = requirements["user_id"]["name_requirements"] %}
+{% set email_reqs = requirements["user_id"]["email_requirements"] %}
+<div class="content shadow" id="login-box">
+    <div id="login-guest">
+     <!-- Add correct URL eventually -->
+        <h1> Sign Up <a target=_blank href="https://submitty.org">
+        <i style="font-style:normal;" class="fa-question-circle"></i>
+        </a> </h1>
```

Comment:
Is the URL available now?

Reply:
Not yet, this next week I will work on creating the docs for this.